Update EQServer(public-release).sh

nazwadi opened the pull request:
Adds chat_opcodes.conf symlink needed by the Universal Chat Service (ucs).

Updates setupini option to add a field in the schema section for loginserver_setting_table, and removes the unused one at the top.

nazwadi committed:
Update EQServer(public-release).sh

Adds chat_opcodes.conf symlink needed by the Universal Chat Service (ucs).

Updates setupini option to add a field in the schema section for loginserver_setting_table, and removes the unused one at the top.

regneq commented on utils/scripts/EQServer(public-release).sh:
should also removed line with mail_opcodes.conf since we do not use those.

nazwadi replied:
Done.  Update ready for review.

nazwadi committed:
Update EQServer(public-release).sh

Removes mail_opcodes.conf symlink and backup based on pull request feedback from @regneq
